extensions/typescript-language-features/package.nls.json

```diff
@@ -5,6 +5,8 @@
 	"configuration.typescript": "TypeScript",
 	"configuration.suggest.completeFunctionCalls": "Complete functions with their parameter signature.",
 	"configuration.suggest.includeAutomaticOptionalChainCompletions": "Enable/disable showing completions on potentially undefined values that insert an optional chain call. Requires TS 3.7+ and strict null checks to be enabled.",
+	"configuration.suggest.includeCompletionsForImportStatements": "Enable/disable auto-import-style completions on partially-typed import statements. Requires using TypeScript 4.3+ in the workspace.",
+	"configuration.suggest.includeCompletionsWithSnippetText": "Enable/disable snippet completions from TS Server. Requires using TypeScript 4.3+ in the workspace.",
```

Comment:
Updated to add an explicit option for snippet-formatted completions, since clients that don’t expect them would insert literal `$1` instead of tab stops.

Reply:
Just as a future note: This is the first example where TS Server uses snippets, but I have a long standing TS issue about using snippets in more places: https://github.com/microsoft/TypeScript/issues/25207